content/pt-br/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -76,40 +78,35 @@ Por exemplo:
   some_counter 0
   ```
 
-Métricas ocultas não são mais publicadas para extração, mas ainda estão disponíveis para uso. Para usar uma métrica oculta, por favor consulte a seção [mostrar métricas ocultas](#mostrar-métricas-ocultas).
+Métricas ocultas não são mais publicadas para extração, mas ainda estão disponíveis para uso.
+Uma métrica ultrapassada se torna uma métrica oculta após um período de tempo, com base em seu nível de estabilidade:
+* Métricas **ESTÁVEL** se tornam ocultas após um mínimo de 3 versões ou 9 meses, o que for mais longo.
+* Métricas **BETA** se tornam ocultas após um mínimo de 1 versão ou 4 meses, o que for mais longo.
+* Métricas **ALFA** podem ser ocultadas ou removidas na mesma versão em que são ultrapassadas.
+
+Para usar uma métrica oculta, por favor consulte a seção [mostrar métricas ocultas](#mostrar-métricas-ocultas).
 
 Métricas excluídas não estão mais disponíveis e não podem mais ser usadas.
 
 ## Mostrar métricas ocultas
 
 Como descrito anteriormente, administradores podem habilitar métricas ocultas por meio de uma _flag_ de linha de comando em um binário específico. Isso pode ser usado como uma saída de emergência para os administradores caso percam a migração das métricas ultrapassadas na última versão.
 
-A _flag_ `show-hidden-metrics-for-version` usa uma versão para a qual você deseja mostrar métricas ultrapassadas nessa versão. A versão é expressada como x.y, onde x é a versão principal e y a versão secundária. A versão de _patch_ não é necessária mesmo que uma métrica possa ser descontinuada em uma versão de _patch_, o motivo é que a política de descontinuação de métricas é executada na versão secundária.
+A _flag_ `show-hidden-metrics-for-version` usa uma versão para a qual você deseja mostrar métricas ultrapassadas nessa versão. A versão é expressada como x.y, onde x é a versão principal e y a versão secundária. A versão de patch não é necessária mesmo que uma métrica possa ser descontinuada em uma versão de patch, o motivo é que a política de descontinuação de métricas é executada na versão secundária.
 
 A _flag_ só pode usar a versão secundária anterior como seu valor. Todas as métricas ocultas no anterior serão emitidas se os administradores definirem a versão anterior como `show-hidden-metrics-for-version`. A versão muito antiga não é permitida porque viola a política de métricas ultrapassadas.
```

Comment:
Esse parágrafo tem alterações que precisam ser feitas também (a pessoa e algumas partes do texto mudaram):
> The flag can only take the previous minor version as its value. If you want to show all metrics hidden in the previous release, you can set the show-hidden-metrics-for-version flag to the previous version. Using a version that is too old is not allowed because it violates the metrics deprecation policy.